List-Id: <gcc-patches.gcc.gnu.org>
List-Archive: <http://gcc.gnu.org/ml/gcc-patches/>
From: Bernhard Reutner-Fischer <rep dot dot dot nop at gmail dot com>
To: gcc-patches at gcc dot gnu dot org
Subject: [PATCH] gcc/: Fix building with isl-0.15.0; includes
Date: Fri, 17 Jul 2015 13:10:26 +0200
Message-Id: <1437131426-27008-1-git-send-email-rep.dot.nop@gmail.com>
In-Reply-To: <1436885112-2174-1-git-send-email-vapier at gentoo dot org>
References: <1436885112-2174-1-git-send-email-vapier@gentoo.org>

In addition to Mike's "[PATCH] gcc: fix building w/isl-0.15" in
https://gcc.gnu.org/ml/gcc-patches/2015-07/msg01162.html
I seem to also need a couple of more includes for isl-0.15.0:

2015-07-15  Bernhard Reutner-Fischer  <aldot@gcc.gnu.org>

	* graphite-blocking.c, graphite-dependences.c,
	graphite-interchange.c, graphite-isl-ast-to-gimple.c,
	graphite-optimize-isl.c, graphite-poly.c,
	graphite-scop-detection.c, graphite-sese-to-poly.c,
	graphite.c: Add missing isl includes.
---
 gcc/graphite-blocking.c          |    1 +
 gcc/graphite-dependences.c       |    1 +
 gcc/graphite-interchange.c       |    1 +
 gcc/graphite-isl-ast-to-gimple.c |    2 ++
 gcc/graphite-optimize-isl.c      |    2 ++
 gcc/graphite-poly.c              |    1 +
 gcc/graphite-scop-detection.c    |    1 +
 gcc/graphite-sese-to-poly.c      |    1 +
 gcc/graphite.c                   |    1 +
 9 files changed, 11 insertions(+)

diff --git a/gcc/graphite-interchange.c b/gcc/graphite-interchange.c
index aee51a8..bcf7f3b 100644
--- a/gcc/graphite-interchange.c
+++ b/gcc/graphite-interchange.c
@@ -24,6 +24,7 @@ along with GCC; see the file COPYING3.
 #include "config.h"
 
 #ifdef HAVE_cloog
+#include <isl/constraint.h>
 #include <isl/aff.h>
 #include <isl/set.h>
 #include <isl/map.h>
diff --git a/gcc/graphite-optimize-isl.c b/gcc/graphite-optimize-isl.c
index 388e25c..2b55783 100644
--- a/gcc/graphite-optimize-isl.c
+++ b/gcc/graphite-optimize-isl.c
@@ -21,7 +21,9 @@ along with GCC; see the file COPYING3.
 #include "config.h"
 
 #ifdef HAVE_cloog
+#include <isl/constraint.h>
 #include <isl/set.h>
+#include <isl/union_set.h>
 #include <isl/map.h>
 #include <isl/union_map.h>
 #include <isl/schedule.h>
diff --git a/gcc/graphite-scop-detection.c b/gcc/graphite-scop-detection.c
index 47e0da0..22fdde3 100644
--- a/gcc/graphite-scop-detection.c
+++ b/gcc/graphite-scop-detection.c
@@ -22,6 +22,7 @@ along with GCC; see the file COPYING3.
 #include "config.h"
 
 #ifdef HAVE_cloog
+#include <isl/constraint.h>
 #include <isl/set.h>
 #include <isl/map.h>
 #include <isl/union_map.h>
diff --git a/gcc/graphite.c b/gcc/graphite.c
index a81ef6a..32f405a 100644
--- a/gcc/graphite.c
+++ b/gcc/graphite.c
@@ -35,6 +35,7 @@ along with GCC; see the file COPYING3.
 #include "config.h"
 
 #ifdef HAVE_cloog
+#include <isl/constraint.h>
 #include <isl/set.h>
 #include <isl/map.h>
 #include <isl/options.h>
-- 
1.7.10.4

